src/controller_pak.h

```diff
@@ -3,35 +3,94 @@
 
 #include "types.h"
 #include "structs.h"
+#include "PR/pfs.h"
+#include "PR/os_cont.h"
 
-extern s32 *D_800DE440;
+#define GAMD 0x47414D44 // Game Data?
```

Comment:
Personally I prefer `#define GAMD 'GAMD'` and the like. It makes it much clearer what the hex actually means.

Reply:
Yeah, I was doing 'GAMD' in the code, but it kept giving warnings and causing other issues because it's technically not proper C code to have string literals with more than one character apparently. It's why I created these defines.

Reply:
Bummer. Maybe just comment that's their ASCII equivalent then?